Description: Override build flags
 Usually the user's build flags, as found in the environment, are
 overridden by the ones chosen by the upstream developer; when building
 Debian packages, the build flags in the environment are provided by
 dpkg-buildflags, and as such must override the ones used in the
 upstream build system.
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2011-11-11
Index: scrotwm.git/linux/Makefile
===================================================================
--- scrotwm.git.orig/linux/Makefile	2011-11-11 10:04:12.761110657 +0100
+++ scrotwm.git/linux/Makefile	2011-11-11 10:04:12.841058350 +0100
@@ -1,8 +1,8 @@
 # $scrotwm: Makefile,v 1.6 2011/08/10 13:44:36 marco Exp $ 
 
-CFLAGS += -Wall -ggdb -I.
-CPPFLAGS += -D_GNU_SOURCE -DSWM_LIB=\"$(LIBDIR)/libswmhack.so.$(LVERS)\"
-LDLIBS += -lX11 -lXrandr -lXtst
+CFLAGS := -Wall -ggdb -I. $(CFLAGS)
+CPPFLAGS := -D_GNU_SOURCE -DSWM_LIB=\"$(LIBDIR)/libswmhack.so.$(LVERS)\" $(CPPFLAGS)
+LDLIBS := -lX11 -lXrandr -lXtst $(LDLIBS)
 
 PREFIX ?= /usr/local
 BINDIR ?= $(PREFIX)/bin
